### ISSUE TITLE
PAP or Perks Not working on CW 2.0 Weapons

### ISSUE THREAD

**Issue by Sazuzaki** [Zet0rz/nzombies#493]
Hello! I'm having an issue with PAP and Weapon Perks (Ex. Speed Cola). When i PAP a weapon, it gives me no attachments. Even when i do it multiple times. And when i get a perk meant for upgrading weapons, for example Speed Cola, it doesn't boost my weapon speed at all. Any help will be appreciated. Thanks!

Extra Info: The weapons i'm using: http://steamcommunity.com/sharedfiles/filedetails/?id=657241323

**Comment by minin43** [comment]
This is a specialized weapon base. It uses a modified CW2.0 weapon base, so it's sort of 2 bases in one. The developer for these weapons is constantly updating the pack, it may not be a good idea to do much with it right now.

**Comment by Punkinator7** [comment]
Have you tried to put it on normally? with C?

**Comment by minin43** [comment]
Attachments should be equipped automatically. But refer back to my first comment.

**Comment by Punkinator7** [comment]
Yeah, but sometimes it wouldn't do it automatically, There is no way that you can't do it if you didn't try it manually. If that doesn't work try unsubscribing to the conflicting add-on, Nothing is perfect my friend.

**Comment by Punkinator7** [comment]
on an unrelated note, when I use CW 2.0 weapons in nZombies, It works fine. I don't experience these problems with MY guns. However, it happens with TFA CS: O1 and 2 weapons on FFGS.

**Comment by minin43** [comment]
My earliest comment:
>This is a specialized weapon base. It uses a modified CW2.0 weapon base, so it's sort of 2 bases in one. The developer for these weapons is constantly updating the pack, *it may not be a good idea to do much with it right now.*
